src/components/Onboarding/Screens/Apps/ConvictionVotingSettings/ConvictionVotingCharts.js

```diff
@@ -0,0 +1,59 @@
+import { GU } from '@1hive/1hive-ui'
+import React, { useMemo } from 'react'
+import {
+  calculateThreshold,
+  toPercentage,
+} from '@/utils/conviction-modelling-helpers'
+import ConvictionTimeChart from './Charts/ConvictionTimeChart'
+import ThresholdRequestedChart from './Charts/ThresholdRequestedChart'
+
+const CHART_HEIGHT = '260px'
+const CHART_WIDTH = '550px;'
+
+const ConvictionVotingCharts = ({
+  decay,
+  maxRatio,
+  minActiveStakePct,
+  requestedAmount,
+  stakeOnProposal,
+  stakeOnOtherProposals,
+  weight,
+}) => {
+  const requestedAmountThresholdPct = useMemo(() => {
+    return toPercentage(
+      calculateThreshold(weight, maxRatio, requestedAmount)
+    ).toFixed(2)
+  }, [maxRatio, requestedAmount, weight])
+
+  return (
+    <div>
+      <div
+        css={`
+          margin-bottom: ${2 * GU}px;
```

Comment:
```suggestion
          margin-bottom: ${4 * GU}px;
```
Maybe it can be a good idea to separate the charts so they don't get that close.